integration/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "npm install && mocha test/*",
+    "test:fast": "mocha test/*"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "chromedriver": "^96.0.0",
+    "geckodriver": "^2.0.4",
+    "mocha": "^9.1.3",
+    "selenium-webdriver": "^4.1.0"
+  },
+  "mocha": {
+    "timeout": 120000
```

Comment:
what is this for?

Reply:
The maximum time for this test to run.
In this case setting the timeout is 120s. This way a test will be given 120s to run, then will quit (timeout).

Reply:
I don't have enough data to choose a proper timeout. And this may not be even necessary if sauce labs handles the timeout stuff. I will update it later according to the real execution time when running on the Sauce Labs.

Reply:
What is the default timeout?

Reply:
I think it's 30s.

Reply:
Can we add a comment here? Also 2 min seems a bit low, can we change it to 5 min?

Reply:
Updated.